pkgs/by-name/sb/sblast/package.nix

```diff
@@ -0,0 +1,45 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+  makeBinaryWrapper,
+  nix-update-script,
+  pulseaudio,
+}:
+buildGoModule rec {
+  pname = "sblast";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "ugjka";
+    repo = "sblast";
+    rev = "v${version}";
+    hash = "sha256-+ZeZ2lohAngfljCa/z9yjCKvQwCMEiwzzPFrpAU8lWA=";
+  };
+
+  vendorHash = "sha256-yPwLilMiDR1aSeuk8AEmuYPsHPRWqiByGLwgkdI5t+s=";
+
+  nativeBuildInputs = [
+    makeBinaryWrapper
+  ];
+
+  postInstall = ''
+    wrapProgram $out/bin/sblast \
+        --suffix PATH : ${lib.makeBinPath [ pulseaudio ]}
```

Comment:
also ffmepg?

```
=> ./results/sblast-x86_64-linux/bin/sblast 
dependency: exec: "ffmpeg": executable file not found in $PATH
```

Reply:
seems like it 😅
fixed, and updated the PR with a passthru test that would have caught this.